galaxy/tools/deps/mulled/mulled_update_singularity_containers.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python
+
+import requests
+from lxml import html
+import subprocess
+import tarfile
+from ruamel.yaml import YAML
+from io import BytesIO
+#from mulled_build import check_output
+from subprocess import check_output
+import logging
+from shutil import copy
+import pickle
+import json
+import argparse
+from jinja2 import Template
+
+from glob import glob
+
+yaml = YAML()
+yaml.allow_duplicate_keys = True
+
+SINGULARITY_DESTINATION = "/data/0/cvmfs/singularity" # file destination for singularity containers
+SINGULARITY_INSTALL = "/opt/singularity/bin/singularity" # location at which singularity is installed, could be something else like /usr/local/bin/singularity
+QUAY_API_ENDPOINT = 'https://quay.io/api/v1/repository'
+
+def get_quay_containers():
+    """
+    Gets all quay containers in the biocontainers repo
```

Comment:
´biocontainers organisation`
Please make biocontainers the default, but offer a commandline argument to make overwrite the default.